### PR TITLE
[PW-7256] Add Bank and Issuing Fund Transfers V3 API

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

Added the models and the following endpoints:

/transactions
/transactions/{id}
/transfers

**Tested scenarios**

Added unit tests based on the open api examples

### CODE REVIEW

**Review by michaelpaul — CHANGES_REQUESTED** [review]
See my previous comment: https://github.com/Adyen/adyen-java-api-library/pull/854#discussion_r1001530658

**Review by michaelpaul — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]

File: src/main/java/com/adyen/service/Transfers.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    public Transfer transfers(TransferInfo transferInfo) throws IOException, ApiException {
+        String jsonRequest = transferInfo.toJson();
+        TransfersResource transfersResource = new TransfersResource(this, "/transfers");
+        String jsonResult = transfersResource.request(jsonRequest);
+        return Transfer.fromJson(jsonResult);
+    }
+
+    public TransactionSearchResponse list() throws IOException, ApiException {
```
Comment (wboereboom) [inline src/main/java/com/adyen/service/Transfers.java:35]: Maybe it is good to rename these. These endpoints are about transactions, whereas the first one is about a Tranasfer.

**Comment by wboereboom** [comment]
MakeFile and Readme should also get updated

File: src/main/java/com/adyen/service/Transfers.java (hunk 1 above)
Comment (wboereboom) [reply src/main/java/com/adyen/service/Transfers.java:35]: @AlexandrosMor can you look into this?

**Pull request merged**